app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -81,6 +81,22 @@ def redirect_to_correct_vendor(vendor, in_hybrid_mobile)
       redirect_to correct_path
     end
 
+    def track_document_request_event(document_request:, document_response:, timer:)
+      document_request_body = JSON.parse(document_request.body, symbolize_names: true)[:config]
+      log_extras = {
+        reference_id: document_response.to_h[:referenceId],
+        vendor: 'Socure',
+        vendor_request_time_in_ms: timer.results['vendor_request'],
+        success: @url.present?,
+        document_type: document_request_body[:documentType],
+      }
```

Comment:
```suggestion
        docv_transaction_token: document_response.to_h.dig(:data, :docvTransactionToken)
      }
```
this attribute is still in the idv_socure_document_request_submitted analytics_event signature but is not passed when creating  the event